.gitignore

```diff
@@ -104,6 +105,7 @@ ReleaseProfile/
 *.dir/
 CPackConfig.cmake
 CPackSourceConfig.cmake
+cmake-build-debug
```

Comment:
For future reference, you can use `.git/info/exclude` to add to have git ignore files that are only relevant to you.

Reply:
Alright, thanks for the tip